### DISCUSSION TITLE
blog/nextra-2

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1255] [Docs Discussions]
# blog/nextra-2

Here are what the new version of Nextra 2 Framework includes.

https://the-guild.dev/blog/nextra-2

**Comment by IlirBajrami** [comment]
Whats the best way to add Nextra to an existing Nextjs project? I see that you recommend using _app.mdx but my existing app is a complete production website and i cant change my _app.tsx to .mdx. Is there a way to add Nextra to that project or i'm forced to create a completely separate app?

**Reply by dimaMachina** [reply]
`_app.mdx` is used for better performance and the smallest bundle size, you can continue to use `_app.tsx`

**Reply by IlirBajrami** [reply]
Does this mean that it will affect my app's overall bundle size and therefore my SEO metrics?

**Reply by dimaMachina** [reply]
feel free to investigate your particular project with a bundle analyzer before/after

**Comment by kunalsinghal** [comment]
Do you have an example repo with MDXRemote?
